modules/gitops-configuring-tls-for-redis-with-autotls-disabled.adoc

```diff
@@ -0,0 +1,235 @@
+// Module is included in the following assemblies:
+//
+// * /cicd/gitops/configuring-secure-communication-with-redis.adoc
+
+[id="gitops-configuring-tls-for-redis-with-autotls-disabled_{context}"]
+= Configuring TLS for Redis with autotls disabled
+
```

Comment:
@ekristova You need to add an introductory sentence or two for this procedure module file. See here for details on the different parts of a procedure module: https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs

Reply:
Hello @Srivaralakshmi thank you for your observations!

As per the guideline for the procedure module, I realized I included part of that information in the introduction of the assembly, more specifically, in that two-point list providing the two methods. Therefore, I made the two points more brief and added the information from there to the introduction of each procedural module, plus I added those special considerations mentioned in the guideline. So PTAL and let me know what you think. 

Also, regarding the "should" issue in the comment below, I changed the wording that in my opinion has a clear meaning and is according to the guidelines. So let me know what you think about that one.

Thank you!